yapapi/engine.py

```diff
@@ -501,52 +501,24 @@ async def _enforce_debit_note_intervals(self, job: "Job", debit_note: DebitNote)
 
     async def _process_debit_notes(self) -> None:
         """Process incoming debit notes."""
+        max_concurrent_notes = 10
```

Comment:
why is this value hardcoded in the middle of the code? if we need such arbitrary values, let's have them defined as `Final` and defined in a constant-like manner in at the top of the file, as is customary with such constants

Reply:
Because `max_number_of_tasks = 5` (used by a different `Semaphore`) was also defined like this in `_find_offers_for_subscription`. Is it better to define both those values as `Final` constants at the top of the file in this PR or to create a separate PR that fixes this?

Reply:
I'd say we should not introduce new issues while adding new code, fixes to the old code can be done in separate pull requests...

Reply:
Done. Changed to `MAX_CONCURRENTLY_PROCESSED_DEBIT_NOTES` and put as a `Final` constant at the top of the file.